BD-3767 Add Optimizely partner docs (#8626)
braze-inc/braze-docs: _docs/_partners/data_and_infrastructure_agility/ab_testing/optimizely.md

---
nav_title: Optimizely
article_title: Optimizely
page_order: 2
description: "This reference article outlines the partnership between Braze and Optimizely that allows you to sync your Braze customer segments, events, and Currents events to Optimizely Data Platform."
alias: /partners/optimizely/
page_type: partner
search_tag: Partner

> layout: dev_guide

Reviewer: ```suggestion
```

The Braze and Optimizely integration is a two-way integration that allows you to:

- Sync your Braze customer segments and events to Optimizely Data Platform (ODP) nightly to enrich Optimizely customer profiles, reports, and segmentation.
- Send Braze Currents events from Braze to Optimizely’s reporting tool.
- Sync ODP customer data and events to Braze to enrich your Braze customer data and trigger Braze messaging based on customer events in ODP.

## Prerequisites

| Requirement                     | Description |
|----------------------------------|-------------|
| Optimizely Data Platform account | An Optimizely Data Platform (ODP) account is required to take advantage of this partnership. |
| Braze REST API key               | A Braze REST API key with the following permissions: `users.track`,`users.export.segments`,`segments.list`,`campaigns.trigger.send`, and `canvas.trigger.send`. |
| Currents                         | To export data back into Optimizely, you need to have Braze Currents set up for your account. |

> | Optimizely URL and Token         | This can be obtained by navigating to your Optimizely dashboard and copying the ingestion URL and token. |
> 

Reviewer: ```suggestion
| Optimizely URL and Token         | This can be obtained by navigating to your Optimizely dashboard and copying the ingestion URL and token. |
{: .reset-td-br-1 .reset-td-br-2 role="presentation" }

```

## Integration

### Step 1: Configure the integration

1. In the **App Directory** of Optimizely Data Platform (ODP), select the **Braze** app and then select **Install App**.
2. Go to the **Settings** tab and in the **Authorization** section, do the following:

>     2.1 Enter Braze **REST API Key**.
>     2.2 Select your Braze **Instance URL**.
>     2.3 Select **Verify API Key**.

Reviewer: Markdown doesn't support sub-step numbering like this. This is currently rendering as a single line 
![image](https://github.com/user-attachments/assets/9a640a83-7abf-45a3-8ec6-8e2905b21894)

```suggestion
    1. Enter Braze **REST API Key**.
    2. Select your Braze **Instance URL**.
    3. Select **Verify API Key**.
```

> 3. Create a Custom Currents Export in Braze using the endpoint and token provided in ODP. This is required to sync Braze events to ODP. <br><br>![Optimizely authorization.][1]

Reviewer: I'm 99% sure this is saying create a Currents connector.

```suggestion
3. Create a [Currents]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/setting_up_currents/) connector in Braze using the endpoint and token provided in ODP. This is required to sync Braze events to ODP. <br><br>![Optimizely authorization.][1]
```
